### ISSUE TITLE
Font awesome is not aligned properly with md-icon

### ISSUE THREAD

**Issue by oshri551** [angular/components#5102]
Bug :
Looks like the font awesome is not working well with md-icon

What is the expected behavior?
icon should be aligned with the text

Plunker template:  [Plunekr](https://plnkr.co/edit/xuq1kFMJUahScaYEWfe2?p=preview)

**Comment by fxck** [comment]
md-icon does not work with *any* icon font out there, that's obvious, so it's expected behaviour and it's up to you to align it

**Comment by jelbourn** [comment]
We should look at this to make sure we're not doing anything weird

**Comment by thekalinga** [comment]
@fxck It should with font awesome if its claiming to work with custom fonts

**Comment by thekalinga** [comment]
I tried with https://cdn.materialdesignicons.com/1.9.32/ aswell & its not aligning with `md-icon` (they claim this to be compatible with material design)

**Comment by fxck** [comment]
You can't optimize it for ALL custom fonts, there's no standard to icons fonts AFAIK.

**Comment by thekalinga** [comment]
Agreed, but would you mind specifying what its compatible with other than stock material icons?

**Comment by thekalinga** [comment]
With material design icons (mdi, not stock material icons), the icon gets displayed but not of the right size & not at the center of the button.

**Comment by willshowell** [comment]
1. `.mat-icon` class has its height and width set to 24px

2. This (I assume) is done because materical icons have a default font-size of 24px. This default is set by the `.material-icons` class via the [icon font](http://google.github.io/material-design-icons/#icon-font-for-the-web).

3. Font awesome defaults its icons to 14px font-size

I think either `.mat-icon` should also set the default font-size to be the same as its height and width OR there should be a brief not in the icon docs about users needing to properly size and position their icons based on their needs

**Comment by thekalinga** [comment]
I tried with mdi aswell. The icon has correct size, but not aligning properly within the md-icon area (I set the icon size to 48px).

They use 24px as default font size. Even there I have the above issue.

So, its better to include an example with custom font that's compatible with material design but not stock icon.

**Comment by willshowell** [comment]
@thekalinga Just to be sure, when you set the icon size to 48px and it was still poorly aligned, did you make the font-size, height, and width all the same?

If so, would you mind showing a plunker with mdi?

**Comment by thekalinga** [comment]
I am unable to create plnkr as the template on the project seems to be broken

I am getting the following error while loading the template. Must be something to do with the recent refactoring w.r.t `cdk`

`Error: (SystemJS) XHR error (404 Not Found) loading http://run.plnkr.co/97mvjrOYfdkUFSc7/@angular/cdk.ts`

**Comment by thekalinga** [comment]
Please check here

https://plnkr.co/edit/VJN3SD68kmQHQvLUZICW?p=preview

**Comment by crisbeto** [comment]
Closing since there hasn't been any activity in a long time.
